app/views/admin/reports/_status.html.haml

```diff
@@ -18,7 +18,7 @@
       - if status.application
         = status.application.name
         ·
-      = link_to ActivityPub::TagManager.instance.url_for(status.proper), class: 'detailed-status__datetime', target: stream_link_target, rel: 'noopener noreferrer' do
+      = link_to ActivityPub::TagManager.instance.url_for(status.proper), class: 'detailed-status__datetime', rel: 'noopener noreferrer' do
```

Comment:
Is there some helper/option which automatically inserts the noopener/noreferrer here, or am I making that up?